db/primary_migrate/20200803211145_backfill_add_acuant_result_to_proofing_costs.rb

```diff
@@ -2,9 +2,5 @@ class BackfillAddAcuantResultToProofingCosts < ActiveRecord::Migration[5.2]
   disable_ddl_transaction!
 
   def up
```

Comment:
Related question: We only need to do anything here because the Ruby class implemlentation wouldn't exist, right? So it's not a problem that other migrations would continue to reference the (later-deleted) tables, since they'd be present at that point in the migration timeline?

https://github.com/18F/identity-idp/blob/d5081da1ed7ae5585349e88de763d5937e9b2b19/db/primary_migrate/20220906112214_add_threatmetrix_to_proofing_cost.rb#L5

Reply:
Yeah exactly, it's because the `ProofingCost` model was removed, so this blew up with a name error.

Another option would have been to redo this as raw SQL because at the time it runs, the table would exist.

I guess we could remove the class, but the timestamp slug would still exist in the `schema_migrations` table? But it's not like we check that very often so it's probably not a big deal.

Do you think it would be clearer to remove entirely?

Reply:
I was mostly curious to improve my understanding of the migrations, but I don't have a strong feeling one way or the other. I think practically speaking this is almost certainly something we won't need. I'd defer to what you think is best, and generally would be fine to keep it as-is.

Reply:
ok! Ill choose to leave it, because I think for future us, it's easier to look back at the history of a mostly empty file that does exist, rather than search history for a file that used to exist